charts/seed-controlplane/charts/packet-cloud-controller-manager/templates/packet-ccm-deployment.yaml

```diff
@@ -0,0 +1,71 @@
+---
+apiVersion: {{ include "deploymentversion" . }}
+kind: Deployment
+metadata:
+  name: packet-cloud-controller-manager
+  namespace: {{ .Release.Namespace }}
+  labels:
+    app: packet-cloud-controller-manager
+    garden.sapcloud.io/role: controlplane
+spec:
+  replicas: {{ .Values.replicas }}
+  selector:
+    matchLabels:
+      app: packet-cloud-controller-manager
+  template:
+    metadata:
+      labels:
+        app: packet-cloud-controller-manager
+{{- if .Values.podAnnotations }}
+      annotations:
+{{ toYaml .Values.podAnnotations | indent 8 }}
+{{- end }}
+    spec:
+      dnsPolicy: Default
+      hostNetwork: true
```

Comment:
In the control plane, probably not. In a classic "I am running on the physical master", perhaps. I can remove it. If it causes issues, we can add it back with a future PR (hopefully _much_ smaller than this one).

Reply:
OK

Reply:
Why is this set? and the dnsPolicy? Not needed i guess